source/commands.txt

```diff
@@ -0,0 +1,114 @@
+.. _vsce-commands:
+
+============================
+MongoDB for VS Code Commands
+============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This document lists the VS Code commands available for
+|vsce|. To open the Command Palette and execute these commands:
+
+.. include:: /includes/list-tables/command-palette-options.rst
+
+Connection Commands
+-------------------
+
+Use these commands to manage connections to your MongoDB deployments.
+
+.. list-table::
```

Comment:
So, a general thought - I have (personally) found that tables like this can be a little hard to parse, especially if they get longer. Also, we can't link to any of them.

I wonder if it would be worth creating a directive for `vsce-command` or something similar so that we can link to these easily.

Alternatively, you can cheat and force these into anchors by doing:

```
- .. _vsce-command-connect:
  :ref:`Connect <vsce-command-connect>`
- Connect to your...

```

You lose the monospacing, but that should be OK (I think you can use xmlrole if you *really* want monospacing).